README.md

```diff
@@ -30,13 +30,26 @@ or e.g. install a snapshot from a
 use `js-mode` as the major mode. To make use of the JS2 AST and the
 packages that integrate with it, we recommend `js2-minor-mode`. See
 the corresponding [instructions in the
-Commentary](https://github.com/mooz/js2-mode/blob/ae9fea81cf9ed80f403c161fde4d8aff1ceb3769/js2-mode.el#L57).
+Commentary](https://github.com/mooz/js2-mode/blob/d9bba5b51097e4915a36f652742588128aca3211/js2-mode.el#L57).
 
 `js-mode` in Emacs 27 includes full support for syntax highlighting
-and indenting of JSX syntax.
+and indenting of JSX syntax. Try it out!
 
-[rjsx-mode](https://github.com/felipeochoa/rjsx-mode/) is an
-alternative option which comes with certain tradeoffs.
+Until [#527](https://github.com/mooz/js2-mode/issues/527) is
+addressed, linting is likely to fail after JSX code. If that bothers
```

Comment:
after => inside/for?

Reply:
I thought that JSX code and the code following it would not be linted correctly, but apparently the parser quickly recovers, so it’s probably only the JSX parts of the code which will show linting errors.  I tried to use clearer language in a recent commit.